lib/registry.js

```diff
@@ -201,12 +201,24 @@ registry.configureModel = function(ModelCtor, config) {
       'must be an array of objects', modelName);
   }
 
+  // Remote methods
+  if (typeof settings.methods === 'object' && settings.methods !== null) {
+    var methods = settings.methods = settings.methods || {};
+    Object.keys(settings.methods).forEach(function(key) {
+      ModelCtor.remoteMethod(key, settings.methods[key]);
```

Comment:
> Please consider the workspace implementation. Lets not make it impractical to implement the generator via the workspace.

Can you please elaborate more why my proposal is impractical for loopback-workspace to implement? Are you afraid of parsing `prototype.{name}` in the workspace?

We can modify the implementation as follows if you think it will make the workspace implementation simpler:

 - `methodName` is mapped to `isStatic:true` unless the config object provides a different value
 - `prototype.methodName` is always mapped to `isStatic:false`, regardless of the value in the config object

```js
// static methods
methods: { "work": { isStatic: true, /*...*/ } }
methods: { "work": { /*...*/ } }

// prototype methods
methods: { "work": { isStatic: false, /*...*/ } }
methods: { "prototype.work": { /*...*/ } }
```

What I really want to avoid is adding more stuff that we will have to rework when fixing #209 and thus causing #209 to introduce even more breaking changes.